docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -10,5 +10,8 @@ There you can find links to upgrade notes for other versions too.
 - *(low priority)* add `product-search-export-products` as a dependency of `build-demo` phing target in your `build.xml`
 if you want to have products data exported to Elasticsearch after `build-demo` target is run ([#824](https://github.com/shopsys/shopsys/pull/824/files))
 
+### Application
+- *(low priority)* if you want to test behavior of cart when no listable product left in cart, implement functional test as it is in [#852](https://github.com/shopsys/shopsys/pull/852)
```

Comment:
i would write there something like `cart with no listable product in cart` but i can understand also this instruction.